Calculo de retenções e implementação da nota fiscal conjugada / nfs-e de Brasilia

mileo opened the pull request:
- Implementa o conceito de retenções;
- Grupo do ISS sobre a nf-e nacional permitindo a emissão de nf-e conjugada e a nfs-e de Brasilia.

mileo committed:
[NEW] Base para o modulo de retenções e inclusão dos campos no cadastro da empresa

mileo committed:
[NEW] Retentenções nos tipos fiscais dos clientes

mileo committed:
[NEW] Campos de retenção nas faturas, mapeamento da retenção por tipo fiscal de parceiro e empresa

mileo committed:
[NEW] Calculo da retencao para pis/cofins/csll por regime de competencia

mileo committed:
[NEW] Calculo retencao ISSQN, CST ISSQN

mileo committed:
[NEW] Implementacao da retencao de tributos

mileo committed:
[NEW] Retenção de ISSQN/PIS/COFINS/CSLL/IR com transmissão de NF-E com PySPED

mileo committed:
[FIX] Correção do calculo de retenção.

mileo committed:
[FIX]

mileo committed:
[FIX] Implementação do calculo WR do IR

danimaribeiro committed:
Migrating modules to new api

mileo committed:
Migração para nova api, e adição de campos no invoice.line

danimaribeiro committed:
Melhorando campos e adicionando atributos readonly

danimaribeiro committed:
Limpeza de código, novas colunas de INSS

danimaribeiro committed:
Obrigatoriedade do campo tipo de serviço, validate_taxes não sobrescreve mais modificações feitas pelo usuário

danimaribeiro committed:
Implementando linhas do account move line para lançar o imposto retido corretamente

danimaribeiro committed:
Mudança na forma de dar baixa nos valores retidos - criado debito e credito ao confirmar a fatura

danimaribeiro committed:
Melhorando o código, remoção de variaveis e código desnecessário

danimaribeiro committed:
Implementado cálculo correto da retenção para notas de entrada e saida - Utilizado campos debito e credito corretamente nas movimentações

danimaribeiro committed:
Atualizando nome Trustcode

danimaribeiro committed:
Mudanças Nota de exportação - Novos campos para o local de despacho e correção de tags do exterior

danimaribeiro committed:
Novo módulo para separar as notas de compra em nota de serviço e nota de produto

hendrixcosta committed:
[MIG] Migrate to v8

hendrixcosta committed:
[FIX] Flake8

sadamo committed:
Correções no sped do l10n_br_account_product para trabalhar com serviços

sadamo commented:
Depends of https://github.com/kmee/PySPED/tree/feature/issqn-fix

sadamo committed:
Adicionadas novas tags referentes ao ISSQN e alterada a tag cMunFG

danimaribeiro commented:
New api:
https://github.com/Trust-Code/l10n-brazil/tree/8.0/l10n_br_account_withholding

sadamo committed:
Migrating withholding functionality from l10n_br_account_withholding to l10n_br_account_product module

sadamo committed:
novos campos e  preenchimento de xml referentes ao ISSQN
Conflicts:
	l10n_br_account_product/models/account_invoice.py
	l10n_br_account_product/models/res_company.py

sadamo committed:
Exigibilidade do ISSQN

sadamo committed:
impostos no tipo de serviço e preenchimento de IR

sadamo committed:
Adicionados impostos por cidade devido as particularidades de cada municipio em relacao ao ISSQN

mileo commented:
@sadamo Por favor faça um rebase com a ultima versão da localização

sadamo committed:
[REM] Removed module l10n_br_account_withholding

sadamo committed:
Remoção de campos relacionados a tag cSitTrib, pois ela não existe mais

sadamo committed:
[FIX] correcao na visao form do tipo de servico

sadamo committed:
[FIX] calculo do valor minimo para retencao de PIS, COFINS e CSLL

sadamo committed:
Valores do CSLL na invoice_line

sadamo committed:
[FIX] Calculo das taxas por municipio

sadamo committed:
Str para preencher tags vISSRet

sadamo committed:
Refatoracao para poder calcular retencao com aliquotas diferentes da aliquota do imposto
